### ISSUE TITLE
"TypeError: (0 , (_inquirer || _load_inquirer(...)).default) is not a function" when initiating a new project

### ISSUE THREAD

**Issue by hollandThomas** [expo/expo#1262]
macOS 10.3.12
node.js 9.4.0
yarn 1.3.2
exp 47.4.4

This is what I get when trying to initiate a new project. It is stuck at 45% now.

```
? Choose a template type: blank
[exp] Downloading project files...
[=============================================                                                       ] 45%(node:9116) UnhandledPromiseRejectionWarning: TypeError: (0 , (_inquirer || _load_inquirer(...)).default) is not a function
    at _callee3$ (/Users/Edgar/universe/dev/exp/src/commands/init.js:111:22)
    at tryCatch (/usr/local/share/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/usr/local/share/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (/usr/local/share/.config/yarn/global/node_modules/regenerator-runtime/runtime.js:114:21)
    at step (/usr/local/share/.config/yarn/global/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /usr/local/share/.config/yarn/global/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (/usr/local/share/.config/yarn/global/node_modules/core-js/library/modules/_export.js:35:28)
    at /usr/local/share/.config/yarn/global/node_modules/babel-runtime/helpers/asyncToGenerator.js:14:12
    at triggerRetryPrompt (/usr/local/share/.config/yarn/global/node_modules/exp/build/commands/init.js:227:18)
    at retryFunction (/Users/Edgar/universe/dev/exp/src/commands/init.js:92:7)
    at Timeout._onTimeout (/xdl/src/Api.js:185:19)
    at ontimeout (timers.js:458:11)
    at tryOnTimeout (timers.js:296:5)
    at Timer.listOnTimeout (timers.js:259:5)
(node:9116) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:9116) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by Xing-He** [comment]
Same issue!!!
- node: v8.9.1
- exp: 47.4.4
- MacOS: 10.13.2

```shell
(node:8773) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: (0 , (_inquirer || _load_inquirer(...)).default) is not a function
(node:8773) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code
```

@hollandThomas Have you solved the problem?

**Comment by hollandThomas** [comment]
@Xing-He not yet. let me know if you get it to work

**Comment by jesseruder** [comment]
Thanks for reporting this! This should be fixed in `exp@48.0.2`.
